app-sso/getting-started/application.hbs.md

```diff
@@ -2,7 +2,7 @@
 
 This topic tells you how to deploy a minimal Kubernetes application that is protected
 by Application Single Sign-On (commonly called AppSSO) by using the credentials
-that [ClientRegistration](client-registration.hbs.md) creates.
+that [tanzu services class-claims](claim-credentials.hbs.md) creates.
 
 ![Diagram of AppSSO's components and how they interact with End-Users and Client applications](../../images/app-sso/appsso-concepts.png)
```

Comment:
**[suggestion]:** Link back to the source Miro board, like [here](https://github.com/pivotal/docs-tap/blob/main/app-sso/concepts/levels-of-consumption.hbs.md?plain=1#L108-L110).

Reply:
Oooh nice